translation/transport.go

```diff
@@ -65,24 +70,66 @@ func ConfigHandler(c *Options) {
 		Methods(http.MethodDelete, http.MethodPut, http.MethodPost)
 }
 
-/* Request Decoding */
+// getPartnerIDs returns the array that represents the partner-ids that were
+// passed in as headers.  This function handles multiple duplicate headers.
+func getPartnerIDs(h http.Header) []string {
+	headers, ok := h[wrphttp.PartnerIdHeader]
+	if !ok {
+		return nil
+	}
+
+	var partners []string
+
+	for _, value := range headers {
+		fields := strings.Split(value, ",")
+		for i := 0; i < len(fields); i++ {
+			fields[i] = strings.TrimSpace(fields[i])
+		}
+		partners = append(partners, fields...)
+	}
+
+	return partners
+}
+
+// getPartnerIDsDecodeRequest returns array of partnerIDs needed for decodeRequest
+func getPartnerIDsDecodeRequest(ctx context.Context, r *http.Request) []string {
```

Comment:
looks perfect! Is the `http.Request` used? I don't think you need that as a parameter 🙂

Reply:
getPartnerIDs uses http.Request